docs/core/testing/mstest-analyzers/mstest0007.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "MSTEST0007: [<TestAttribute>] can only be set on methods marked with [TestMethod]"
```

Comment:
Just wondering if we should have `Test attributes .... on test methods`

Reply:
This is the way it was originally (more or less, it was `Ensure test attributes ...`) but then there was a suggestion from you to change it to `[<TestAttribute>] ...`. Would you like to change it back ?

Reply:
Updated as per our offline sync.